temporalcli/commands.workflow.go

```diff
@@ -187,33 +186,53 @@ func (c *TemporalWorkflowTerminateCommand) run(cctx *CommandContext, _ []string)
 	return nil
 }
 
-func (c *TemporalWorkflowUpdateCommand) run(cctx *CommandContext, args []string) error {
-	cl, err := c.Parent.ClientOptions.dialClient(cctx)
+func (c *TemporalWorkflowUpdateStartCommand) run(cctx *CommandContext, args []string) error {
+	return workflowUpdateHelper(cctx, c.Parent.Parent.ClientOptions, c.PayloadInputOptions, c.UpdateOptions, client.WorkflowUpdateStageAccepted)
+}
+
+func (c *TemporalWorkflowUpdateExecuteCommand) run(cctx *CommandContext, args []string) error {
+	return workflowUpdateHelper(cctx, c.Parent.Parent.ClientOptions, c.PayloadInputOptions, c.UpdateOptions, client.WorkflowUpdateStageCompleted)
+}
+
+func workflowUpdateHelper(cctx *CommandContext,
+	clientOpts ClientOptions,
+	inputOpts PayloadInputOptions,
+	updateOpts UpdateOptions,
+	waitForStage client.WorkflowUpdateStage,
```

Comment:
Didn't we decide for start that we are going to require this be provided by users in hopes that later we can default this to durable admitted?

Reply:
Oh that's right! With the need to add new subcommands I completely forgot about that. Good catch.